### PR TITLE
Update and improve TCA

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Configuration/TCA/tx_blogexample_domain_model_blog.php [hunk 1]
```diff
@@ -233,5 +234,8 @@
                 --linebreak--,
                 fe_group',
         ],
+        'language' => [
+            'showitem' => 'sys_language_uid, l10n_parent,',
```
Comment (froemken) [inline Configuration/TCA/tx_blogexample_domain_model_blog.php:238]:
```suggestion
            'showitem' => 'sys_language_uid, l10n_parent',
```

Comment (froemken) [inline Configuration/TCA/tx_blogexample_domain_model_blog.php:238]: What about `l10n_diffsource`?

File: Resources/Private/Language/locallang_db.xlf [hunk 2]
```diff
@@ -21,7 +21,7 @@
 			<trans-unit id="tx_blogexample_domain_model_blog.administrator" resname="tx_blogexample_domain_model_blog.administrator">
 				<source>Administrator</source>
 			</trans-unit>
-			<trans-unit id="tx_blogexample_domain_model_blog.administrator" resname="tx_blogexample_domain_model_blog.administrator.description">
+			<trans-unit id="tx_blogexample_domain_model_blog.administrator.description" resname="tx_blogexample_domain_model_blog.administrator.description">
```
Comment (froemken) [inline Resources/Private/Language/locallang_db.xlf:24]: Good point. Thx.

**Review by froemken — CHANGES_REQUESTED** [review]

File: Configuration/TCA/tx_blogexample_domain_model_blog.php (hunk 1 above)
Comment (linawolf) [reply Configuration/TCA/tx_blogexample_domain_model_blog.php:238]: l10n_diffsource is passthrough so not nessesary to add it to a palette

**Review by froemken — APPROVED** [review]
I got tears with such line:
`public ?ObjectStorage $logo = null;`
But I remember you need such lines as example, right?

**Pull request merged**